Patch: 20_qmail_to_mailman.debian.patch
Author: Tollef Fog Heen <tfheen@debian.org>
Fix some paths in the qmail_to_mailman wrapper (Closes: #217573)
Index: mailman-2.1.12/contrib/qmail-to-mailman.py
===================================================================
--- mailman-2.1.12.orig/contrib/qmail-to-mailman.py	2009-03-02 20:41:54.000000000 +0100
+++ mailman-2.1.12/contrib/qmail-to-mailman.py	2009-03-02 20:50:03.000000000 +0100
@@ -28,12 +28,12 @@
 #
 # INSTALLATION:
 #
-# Install this file as ~mailman/qmail-to-mailman.py
+# Install this file as ~mailman/bin/qmail-to-mailman.py
 #
 # To configure a virtual domain to connect to mailman, create these files:
 #
 # ~mailman/.qmail-default
-# |preline @PYTHON@ @prefix@/mail-in.py
+# |preline /usr/bin/python /usr/lib/mailman/bin/mail-in.py
 #
 # /var/qmail/control/virtualdomains:
 # DOMAIN.COM:mailman
@@ -64,7 +64,7 @@
         sys.exit(100)
 
     local = string.lower(local)
-    local = re.sub("^mailman-","",local)
+    local = re.sub("^" + MailmanUser + "-","",local)
 
     names = ("root", "postmaster", "mailer-daemon", "mailman-owner", "owner",
              "abuse")
